clients/cloud/kafkacat/kafkacat-example.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -eu
+
+# Set topic name
```

Comment:
how about using KAFKACAT_CONFIG instead of passing -F? `export KAFKACAT_CONFIG=~/.ccloud/config`

Reply:
I added a comment about this in the example but I don't want the example to actually use it.  The reason is the copy/pasters will forget about the export